src/CFITSIO.jl

```diff
@@ -1043,16 +1375,16 @@ function fits_read_subset(
         (
             Ptr{Cvoid},
             Cint,
-            Ptr{Clong},
-            Ptr{Clong},
-            Ptr{Clong},
+            Ref{Clong},
+            Ref{Clong},
+            Ref{Clong},
```

Comment:
Why this change?

Reply:
My understanding was that `Ref` is equivalent to `Ptr` except it's an object that has its memory managed by Julia, so there are no risks of memory being freed during the `ccall`. Is this not a correct usage of `Ref`? Or is it unnecessary as these are arguments to the function anyway?

Reply:
From the Julia manual:
> When passed as a `ccall` argument (either as a `Ptr` or `Ref` type), a Ref object will be converted to a native pointer to the data it references. For most T, or when converted to a `Ptr{Cvoid}`, this is a pointer to the object data. When T is an `isbits` type, this value may be safely mutated, otherwise mutation is strictly undefined behavior.

So at first glance it doesn't seem like there is a difference between `Ptr` and `Ref`? I'm not sure, though.